### ISSUE TITLE
mafft qinsi missing dependencies 

### ISSUE THREAD

**Issue by laure182** [bioconda/bioconda-recipes#7321]
bioconda recipe of mafft version 7.245 and 7.310 returns the following error using the qinsi flavor :
mafft-qinsi --nuc --ep 0.123 --op 1.53   myinputfile 

= Install MXSCARNA ======================================================
MXSCARNA (Tabei et al. BMC Bioinformatics 2008 9:33) is required.
Please 'make' at the 'extensions' directory of the MAFFT source package,
which contains the modified version of MXSCARNA.
http://mafft.cbrc.jp/alignment/software/source.html 
==========================================================================

The make in the extensions directory is not in the conda recipe.

**Comment by bgruening** [comment]
ping @caleb-easterly

**Comment by krinsman** [comment]
Yeah I had to compile mafft-with-extensions from source to use mafft-qinsi too. I guess it is installed without extensions because the extensions include, besides MXSCARNA and probconsRNA and Rfold (which as far as I can tell are not Bioconda packages), also viennaRNA (which _is_ a Bioconda package https://anaconda.org/bioconda/viennarna).

So the "proper conda way" for making a package that install mafft-qinsi would probably require first making packages for MXSCARNA, probconsRNA, and Rfold. Which I guess no one has had the bandwidth/desire to do yet. Also it seems worth noting that the MXSCARNA link on the MAFFT webpage appears to be out of date, and appears to now be here https://www.ncrna.org/softwares/mxscarna/dl/ on the same website as Rfold https://www.ncrna.org/softwares/rfold/.

> **TL;DR** my guess is that the reason a mafft-qinsi package hasn't been made yet is that the tarball for it on the MAFFT website doesn't "play nice" with the already existing probcons and viennaRNA packages, and that to address this would require the creation of several additional other packages besides mafft-qinsi and linking them together correctly.

(Note that there isn't a probconsRNA bioconda package even though there is a probcons, trained for protein data, bioconda package https://anaconda.org/bioconda/probcons . Since the authors of probconsRNA seem to consider it experimental, they seem not to have modified the documentation aspects of probconsRNA from probcons proper much, making it confusing to determine which is which when looking only at the source code, and when I compiled probconsRNA I had to manually rename the binary to not conflict with the probcons binary from bioconda also in my PATH.)
